### ISSUE TITLE
F0 Converter for P - loss function values

### ISSUE THREAD

**Issue by rishabhjain16** [auspicious3000/autovc#88]
I am trying to replicate your work. I am currently making F0 converter model for P checkpoint generation. I am stuck at loss calculation. 

I see when I use F0_Converter model to generate P, I get a 257 dimension one-hot encoded feature P. 

Demo.ipynb
```
f0_pred = P(uttr_org_pad, f0_trg_onehot)[0]
f0_pred.shape
> torch.Size([192, 257])
```
I wanted to ask you when training the F0 converter model, what is the value that you are using to calculate the loss? 

I tried using the following value but I am not sure if that is the right way. 
This is what I am doing to generate f0_pred and to calculate the loss:

f0_pred = self.P(x_real_org,f0_org_intrp)[0]
p_loss_id = F.mse_loss(f0_pred,f0_org_intrp,reduction='mean')

I just want to know if I am on the right track. 
Can you help me out here @auspicious3000

**Comment by rishabhjain16** [comment]
Sorry wrong Repo. My Bad
